cfgov/cfgov/urls.py

```diff
@@ -385,11 +385,11 @@ def empty_200_response(request, *args, **kwargs):
     re_path(r'^_status/', include_if_app_enabled('watchman', 'watchman.urls')),
 
     re_path(
-        r'^consumer-tools/financial-well-being/',
+        r'^(?i)consumer-tools/financial-well-being/',
```

Comment:
Wait, but don't these need to be at the end of the line? My impression from #5640 was that we'd need to add this to the individual URL patterns in [`wellbeing/urls.py`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/wellbeing/urls.py)?

Reply:
Agreed and fixed